pkg/gossip/gossip.proto

```diff
@@ -130,7 +130,7 @@ message InfoStatus {
   map<string, Info> infos = 1 [(gogoproto.nullable) = false];
   ClientStatus client = 2 [(gogoproto.nullable) = false];
   ServerStatus server = 3 [(gogoproto.nullable) = false];
-  Connectivity connectivity = 4 [(gogoproto.nullable) = false];
+  Connectivity connectivity = 4 [(gogoproto.nullable) = false, deprecated = true];
```

Comment:
Not relevant to current work. Relates to following PR: https://github.com/cockroachdb/cockroach/pull/89613